lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -587,7 +587,7 @@ def add_echeck(xml, check, options = {})
 
       def add_credit_card(xml, credit_card, options)
         xml.tag! :AccountNum, credit_card.number if credit_card.is_a?(CreditCard)
-        xml.tag! :Exp, expiry_date(credit_card) if credit_card.is_a?(CreditCard)
+        xml.tag! :Exp, options[:override_exp_date] || expiry_date(credit_card) if credit_card.is_a?(CreditCard)
```

Comment:
a group of lines (before and after this one) have the same if statement: `credit_card.is_a?(CreditCard)`
I suggest doing just one, is this possible?

Reply:
I left lit like this because of currency, since it can be used by other payment method